stdlib/LinearAlgebra/test/triangular.jl

```diff
@@ -538,4 +538,15 @@ end
     end
 end
 
+@testset "special printing of Lower/UpperTriangular" begin
+    @test sprint(show, MIME"text/plain"(), LowerTriangular(2ones(Int,3,3))) ==
+        "3×3 LinearAlgebra.LowerTriangular{Int64,Array{Int64,2}}:\n 2  ⋅  ⋅\n 2  2  ⋅\n 2  2  2"
```

Comment:
Test failure on 32-bit, need to interpolate `$Int` to the strings here rather than hardcoding `Int64`.